js/piwik.js

```diff
@@ -2468,6 +2471,12 @@ if (typeof window.Matomo !== 'object') {
                 // for outlinks or referrers
                 url = removeUrlParameter(url, configVisitorIdUrlParameter);
 
+                if (configExcludedQueryParams.length > 0) {
+                    for (var i = 0; i < configExcludedQueryParams.length; i++) {
```

Comment:
Our JSLint complains about this, as variable declarations should be at the beginning. So simply remove the `var` here and add `i` at the beginning so its `var targetPattern, i;`

Reply:
I'll get that variable declaration fixed up and take a look at adding those tests. Thanks for the hints..